app/Providers/AppServiceProvider.php

```diff
@@ -31,8 +30,9 @@ public function boot()
      */
     public function register()
     {
-        if ($this->app->environment('local', 'testing')) {
-            $this->app->register(DuskServiceProvider::class);
-        }
+        // Comment out as we are removing Dusk from the repository
+        // if ($this->app->environment('local', 'testing')) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS125)